fragments/master.sh

```diff
@@ -25,6 +25,16 @@ if ( ! ps -ef | grep "/usr/bin/docker" | grep -v 'grep' &> /dev/null ); then
     exit 1
 fi
 
+RESTART_POLICY="no"
+START_MODE=$1
+
+if [[ -n ${START_MODE} && ${START_MODE} == "install" ]]; then
+   echo "Start k8s cluster in installation mode"
+   RESTART_POLICY="always"
+else
+   echo "Start k8s cluster in run mode"
```

Comment:
For consistency reasons  I would suggest to place RESTART_POLICY here. Otherwise the code feels a bit "unbalanced"